packages/field-label/README.md

```diff
@@ -1,32 +1,58 @@
-## Description
+## Overview
 
-An `<sp-field-label>` provides accessible labelling for form elements. Use the `for` attribute to outline the `id` of an element in the same DOM tree to which it should associate itself.
```

Comment:
Let's keep this line.

Reply:
im going to combine them in to a short paragraph as the new line comes from design guidance too.